docs/2.7.0/docs/daml-finance/reference/patterns.rst

```diff
@@ -0,0 +1,29 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Patterns
+########
+
+This page some common patterns used in the Daml Finance library.
+
+.. _getview:
+
+View the data of a contract
```

Comment:
I feel this is specific to interfaces, so what about renaming the section to something like

```suggestion
View of an interface contract and the `GetView` choice
```